server/core/src/dto/expenses.rs

```diff
@@ -47,3 +47,36 @@ pub struct NewPredefinedExpenseRequest {
     pub currency_type_id: Id,
     pub recurrence_type_id: Id,
 }
+
+pub type ExpenseResponse = Vec<ExpenseWithTransactions>;
```

Comment:
Aliases are usually named after the function which returns them, so this one should be `GetExpensesResponse`.

Reply:
Deleting this file now.